weblate_web/templates/hosting.html

```diff
@@ -652,15 +652,15 @@ <h2 class="section-title crypto payment-conditions">{% trans "Payment" %}</h2>
             </div>
             <h2 class="section-title crypto payment-conditions" id="strings">{% trans "Source strings" %}</h2>
             <div class="page-desc small">
-                <p>{% trans "Pricing is based on number of source strings and target languages. The source string is a text unit defined in a translation format, it can be a word, sentence or paragraph." %}</p>
+                <p>{% trans "Pricing is based on the number of source strings and target languages. A source string is a text unit defined in a translation format, it can be a word, sentence or paragraph." %}</p>
             </div>
 
             <div class="libre-hosting">
-              <h2>{% trans "Weblate supports libre projects" %}</h2>
+              <h2>{% trans "Weblate proudly supports libre projects" %}</h2>
               <p>
-                {% blocktrans with url="https://hosted.weblate.org/hosting/" %}We proudly support open source projects across the globe. If you feel like being supported by Weblate will help your libre project, feel free to start free trial and <a href="{{ url }}">let us know</a>.{% endblocktrans %}
-                {% blocktrans %}For such projects, we would activate <strong>Libre plan</strong> for free.{% endblocktrans %}
-                {% blocktrans %}It has same limits as <strong>Advanced plan</strong>, but only allows public projects.{% endblocktrans %}
+                {% blocktrans with url="https://hosted.weblate.org/hosting/" %}If you feel like being supported by Weblate will help your libre project, <a href="{{ url }}">reach out</a>{% endblocktrans %}
```

Comment:
Even though the process is not yet there right, the strings are already based on the  new workflow for libre hosting which should be available shortly. Users will create trial, setup their translations and will ask for approval for the libre plan. That will make it both faster (users can start using Weblate without waiting for us) and less work for us.

Reply:
I don't see it relevant to explain that opting for the libre plan is for libre projects. The qualifier "libre" is already the reason it is gratis. All of what happens is what the user does anyway. It is better to go through the process than reading about it beforehand? "Trial" is not a good name for the libre plan, as it suggests it is some kind of beta. I opted for "set up your project", which will work now and later.

Reply:
It will be trial period same as for commercial hosting. For libre hosting the next step will be manual approval for libre plan.

Reply:
That, along with how much time it takes, isn't really something the user needs to know about.
There is one way to do it, and that is how it is done. No need to explain.